classes/PrestaShopAutoload.php

```diff
@@ -286,9 +286,9 @@ protected function getClassesFromDir($path, $hostMode = false)
                 } elseif (substr($file, -4) == '.php') {
                     $content = file_get_contents($rootDir . $path . $file);
 
-                    $namespacePattern = '[\\a-z0-9_]*[\\]';
+                    $namespacePattern = '[\\\\a-z0-9_]*[\\\\]';
```

Comment:
We're in a quote context.
```php
echo '\t';  // => '\t'
echo "\t"; // => '	'
```
Can you give us an example where it fails? :thinking:

Reply:
PHP 7.3 uses newer POSIX regex engine and regexes like ~[\d-_]~ became invalid. The fixed regex may became invalid someday too, so I fixed it. - as character is safe to use only as regular range operator or when it is placed the most left or the most right in the character group.